src/Adapter/StockManager.php

```diff
@@ -133,6 +133,11 @@ public function updatePhysicalProductQuantity($shopId, $errorState, $cancellatio
      */
     private function updateReservedProductQuantity($shopId, $errorState, $cancellationState, $idProduct = null, $idOrder = null)
     {
+        $statesToIgnore = [
+            (int) $errorState,
+            (int) $cancellationState,
+            (int) (new ConfigurationAdapter())->get('PS_OS_REFUND'),
```

Comment:
Could you move `ConfigurationAdapter` to the constructor ?

Reply:
Agreed with @Progi1984 

@timactive Dependency injection is a must-have, can you please inject the Adapter and not instantiate it here?